packages/server/test/util/factories.ts

```diff
@@ -46,7 +46,11 @@ export const OfficeHourFactory = new Factory(OfficeHourModel)
 
 export const CourseFactory = new Factory(CourseModel)
   .attr('name', 'CS 2500')
-  .attr('icalURL', 'http://hi.com')
+  // calendar is owned by sandboxneu@gmail.com
+  .attr(
+    'icalURL',
+    'https://calendar.google.com/calendar/ical/t6lu2pic7u9otrbpkuk26sl34g%40group.calendar.google.com/public/basic.ics',
```

Comment:
i should probably test if it works using an outlook url as well

Reply:
i tested with outlook and it seemed good, but feel free to test using your own calendars as well